.gitignore

```diff
@@ -1,25 +1,12 @@
 *~
 extracted
 source/extracted
-replays
 .dub
 3d
-SacEngine-*
-SacEnginePi*
-SacEngineA*
-SacEngine.*.txt
-ldc2*
 dub_platform*
-settings.txt
 *.scr
-*.rcp
 zeroip.txt
 *.7z
-fonts
-data
-maps
-data-orig
-maps-orig
 *.zip
 *.exe
 *.dll
```

Comment:
i guess these could be moved to windows dependencies ... ? leave them until a better suggestion comes along.

Reply:
exe, dll, dylib and so could be in a category `# ignore binaries` if you like.

Reply:
alright. will do.

Reply:
resolving.